### ISSUE TITLE
extra matplotlib plots displayed in the Output panel when no PBs are available

### ISSUE THREAD

**Issue by rlisahuang** [UCSD-PL/vscode#45]
To replicate (you do need to keep the comments at the end to replicate the faulty behavior):

```py
import itertools
import matplotlib.pyplot as plt

# plt.ion()


data = [180,250,130,50]


# plot a bar chart
plt.bar(range(len(data)), data)
# plt.show()

plt.xticks(range(len(data)), ['Fall', 'Winter', 'Spring', 'Summer'])
# plt.show()

# change color of each bar
# colors = ['red', 'green', 'yellow', 'blue']
# for i, d in enumerate(data):
#     plt.gca().patches[i].set_color(colors[i])
#     # plt.show()

# plt.show()
```

Expected output (two plots):
<img width="516" alt="Screenshot 2023-03-09 at 6 56 28 PM" src="https://user-images.githubusercontent.com/29112035/224211753-ebb00f1b-e6e1-4a32-8e2c-2aecc2f82918.png">

Actual output (four plots, with the second plot showing up three times)
<img width="266" alt="Screenshot 2023-03-09 at 6 57 16 PM" src="https://user-images.githubusercontent.com/29112035/224211871-6187453a-2393-406e-87c2-1d7150bd9354.png">

**Comment by KasraF** [comment]
@rlisahuang Why is the expected output two plots? All calls to `plt.show()` are commented out in the code example.

**Comment by rlisahuang** [comment]
@KasraF Ah right. No plots should be there at all because no `plt.show()` is called.

**Comment by slerner** [comment]
Can we close this? Or is there still an issue to address?

**Comment by rlisahuang** [comment]
@KasraF looks like it has been fixed in 38659aa67f134a334348203398e5a6c637b53bb6? Please close if that is the case.

**Comment by KasraF** [comment]
"Fixing" this requires detecting _actual_ calls to `plt.show` in the code, which we do not do. However, as a temporary hack, I used a slightly more involved regex to check if `plt.show(` appears in comments/strings, or if it's actually in the code. So this particular instance of this bug is fixed. Closing this issue.
